src/ui/components/pages/otherAccounts.tsx

```diff
@@ -106,7 +108,13 @@ export function OtherAccountsPage({ setTab }: Props) {
             className={styles.addAccountButton}
             data-testid="addAccountButton"
             type="button"
-            onClick={() => setTab(PAGES.IMPORT_FROM_ASSETS)}
+            onClick={() => {
+              background.showTab(
+                `${window.location.origin}/accounts.html`,
+                'accounts'
+              );
+              dispatch(resetTab(PAGES.ROOT));
```

Comment:
Is it necessary to change screen after opening a tab?

Reply:
Yeah, this is necessary so that the tests pass and do not violate the user flow